.github/workflows/deploy.yml

```diff
@@ -14,10 +14,10 @@ jobs:
       - name: Checkout Source Files
         uses: actions/checkout@v2
 
-      - name: Use Node.js 16
+      - name: Use Node.js 18
         uses: actions/setup-node@v2
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
@jdwilkin4 @naomi-lgbt @scissorsneedfoodtoo  guys, the latest node version is 20, should we still stick with 18 ?

Reply:
thanks @JoyShaheb !
totally good point!